controllers/user.controller.js

```diff
@@ -0,0 +1,63 @@
+// controllers/userController.mjs
+import pool from '../dbConfig.mjs';
+
+export const createUser = async (req, res) => {
+  try {
+    const { username, age, email } = req.body;
+    const query = `
+      INSERT INTO users (username, age, email)
+      VALUES ($1, $2, $3)
+      RETURNING *;
+    `;
+    const values = [username, age, email];
+    const result = await pool.query(query, values);
+    res.json(result.rows[0]);
+  } catch (error) {
+    res.status(500).json({ error: 'An error occurred.' });
```

Comment:
Errors should be descriptive enough for the client. Maybe we can give much more information in the error message? 
```suggestion
    res.status(500).json({ error: 'Failed to create new user.' });
```
And also possibly log the error with a `console.error()` before returning the json so we can tell what the problem was?